src/domain/orders/edit/modal.tsx

```diff
@@ -418,7 +418,7 @@ function OrderEditModalContainer(props: OrderEditModalContainerProps) {
   }, [activeOrderEditId])
 
   const onClose = () => {
-    setActiveOrderEdit(undefined)
+    // setActiveOrderEdit(undefined) -> context will unset active edit after flag toggle
```

Comment:
question: Do you think we should keep this in?

Reply:
Since the flow is a bit unconventional I think it's ok to leave this comment here for clarification